proto/core/publish.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+cd ./proto/core
+cd ./proto-ts || exit
+
+v=$(npm view @injectivelabs/core-proto-ts version)
+echo "current package version: $v"
+
+v1="${v%.*}.$((${v##*.}+1))"
+echo "new package version: $v1"
+
+npm version $v1
+npm publish
+
+## OLD PACKAGE DEPRECATED
+## Publish chain-api
+# cd ./../chain-api || exit
+
+# v=$(npm view @injectivelabs/chain-api version)
+# echo "current package version: $v"
+
+# v1="${v%.*}.$((${v##*.}+1))"
+# echo "new package version: $v1"
+
+# npm version $v1
+# npm publish
```

Comment:
nit: can we remove this?

Reply:
I wanted to keep it comment in case some external developer still uses these old packages so we can easily uncomment and generate those as well

Reply:
removed!